src/game_timer.cpp

```diff
@@ -0,0 +1,52 @@
+// Header
+#include "game_timer.hpp"
+
+// stlib
+#include <vector>
+#include <algorithm>
```

Comment:
do we actually use either of these `#include`s? I'm guessing they have protections so it won't break our program to include them multiple times but I don't if that adds additional overhead or not